tests/testthat/test-MultipleLambdas.R

```diff
@@ -1,5 +1,5 @@
 context("MultipleLambdas")
-
+data("soccer", package = "glmmLasso")
```

Comment:
@jaredlander this line gets the data from it's original location, so it doesn't need to be held in both packages